tests/InputFiles/GeneralizedHarmonic/KerrSchild.yaml

```diff
@@ -54,7 +54,7 @@ EventsAndTriggers:
   ? EveryNSlabs:
       N: 5
       Offset: 2
-  : - Interpolate
+  : - Horizon
```

Comment:
Is it possible to set the option name string for the event so it is more obvious this is an interpolation?

Reply:
In particular, it would be nice to disambiguate between interpolating to some horizon-like strahlkorper, vs. performing an iterative horizon find.

Reply:
Yes, you can set the name to whatever you want.  It uses the name of the InterpolationTargetTag (or a name function inside InterpolationTargetTag), which you can set to whatever you want. Maybe I shouldn't have picked the name 'Horizon'.  Maybe 'InterpolateToKerrHorizon'?

Reply:
Personally I find this suggestion much clearer 👍

Reply:
was this resolved? The input file still says `Horizon`